src/app/services/searchResults.ts

```diff
@@ -178,10 +178,10 @@ const siteShortcuts: SearchShortcut[] = siteSpecific([
         type: SEARCH_RESULT_TYPE.SHORTCUT
     }, {
         id: "my_gameboards",
-        title: "My Gameboards",
-        terms: ["gameboards", "gameboard", "my gameboards"],
-        summary: "View your saved gameboards.",
-        url: "/my_gameboards",
+        title: "My Question Decks",
+        terms: ["gameboards", "gameboard", "my gameboards", "question deck", "question decks", "my question decks"],
```

Comment:
Maybe we could add "boards" and "decks" too?
```suggestion
        terms: ["gameboards", "gameboard", "my gameboards", "boards", "question deck", "question decks", "my question decks", "decks"],
```